pyairtable/api/abstract.py

```diff
@@ -9,22 +9,25 @@
 
 from .params import to_params_dict
 
+DEFAULT_MAX_RECORDS_PER_REQUEST = 10
+
 
 class ApiAbstract(metaclass=abc.ABCMeta):
     VERSION = "v0"
     API_BASE_URL = "https://api.airtable.com/"
     API_LIMIT = 1.0 / 5  # 5 per second
     API_URL = posixpath.join(API_BASE_URL, VERSION)
-    MAX_RECORDS_PER_REQUEST = 10
 
     session: requests.Session
-    tiemout: Optional[Tuple[int, int]]
+    timeout: Optional[Tuple[int, int]]
 
-    def __init__(self, api_key: str, timeout=None):
+    def __init__(self, api_key: str, timeout: Optional[int]=None,
+                 max_records_per_request: Optional[int]=DEFAULT_MAX_RECORDS_PER_REQUEST):
```

Comment:
Perhaps `timeout` and `max_records_per_request` should be explicitly keyword-only?
This is already the case in the `Table`'s __init__ method, and following through to here would provide a clearer extension path and also cleaner/DRYer code in the subclasses.
```suggestion
    def __init__(
        self,
        api_key: str,
        *, 
        timeout: Optional[int]=None,
        max_records_per_request: Optional[int]=DEFAULT_MAX_RECORDS_PER_REQUEST
    ):
```

Reply:
+1 I also prefer this and wish I had added from the beginning 
However, we should consider adding this would be a breaking change

Reply:
Good point. I doubt anyone is passing the timeout argument as a positional parameter, but better to be nice about it :P